tests/deprecated_api/test_remove_1-7.py

```diff
@@ -255,3 +256,37 @@ def test_v1_7_0_deprecate_lightning_distributed(tmpdir):
         from pytorch_lightning.distributed.dist import LightningDistributed
 
         _ = LightningDistributed()
+
+
+def test_v1_7_0_deprecate_on_post_move_to_device(tmpdir):
+    class WeightSharingModule(BoringModel):
```

Comment:
let's use the same terminology here too:
```suggestion
    class ParameterSharingModule(BoringModel):
```